feature-group-definitions/color-scheme.yml

```
@@ -0,0 +1,8 @@
+name: color-scheme
+description: The `color-scheme` CSS property defines which color schemes (light or dark) an element can be rendered in and prevents automatic dark mode adjustments by the browser.
```

A few changes for brevity and possibly clarity. Specifically:

- I initially read "rendered in" as meaning "(don't) render at all"
- The `only` bit is optional

```suggestion
description: The `color-scheme` CSS property sets which color schemes (light or dark) an element uses and may prevent automatic dark mode adjustments by the browser.
```

Also, this reminded me of something: I once wrote [a little style guide for writing short descriptions](https://github.com/mdn/short-descriptions/blob/master/writing-short-descriptions.md)! It was for CSS only (at least to start), so it's a little over-specific for us to adopt in full, but it does suggest to me some changes here, specifically:

> Short descriptions should avoid the use of words like "specifies", "defines", or "determines" when the word "sets" is just as good and results in no loss of meaning.

---

I like these edits, applying without further debate (for once) 😄